app/controllers/patients_controller.rb

```diff
@@ -17,6 +17,7 @@ def show
     redirect_to(root_url) && return unless current_user.can_view_patient?
 
     @patient = current_user.get_patient(params.permit(:id)[:id])
+    @jurisdiction = @patient.jurisdiction
```

Comment:
Little confused why this is apparently being hit while GETting sara.png, but I'm seeing this in the output when I load a patient page:

![image](https://user-images.githubusercontent.com/8235974/116094932-f2cf7a00-a675-11eb-9abb-b593e23d4965.png)

Reply:
No longer seeing this since your last push, so I guess it's a non-issue

Reply:
seems like the request is `GET /patients/sara.png` which is causing the controller to try to authenticate the user. maybe the `href` for the logo is doing some kind of relative path?

Reply:
There was a relative path issue, which was fixed friday. Guessing the "before" was before Tim rebased over the latest.